package.json

```diff
@@ -158,7 +158,7 @@
     "pouchdb-quick-search": "^1.3.0",
     "query-string": "^6.2.0",
     "r2-lcp-js": "^1.0.6",
-    "r2-navigator-js": "^1.0.16",
```

Comment:
Just wondering: is there a specific reason for using `r2-navigator-js` version `1.0.17` in both `package-lock.json` and `package.json`? The latest version is `1.0.18`, see this PR (still pending approval/merge, but I do not know why):
https://github.com/readium/readium-desktop/pull/202/files

Without the latest version, you are missing out on useful "reading location" reporting features (which are crucial for a "progress" interactive timeline / scrubber):
https://github.com/readium/r2-navigator-js/blob/develop/CHANGELOG.md#1018
...documented here:
https://github.com/readium/r2-navigator-js/blob/develop/README.md#reading-location-linking-with-locators